src/main/webapp/WEB-INF/web.xml

```diff
@@ -4,6 +4,15 @@
     xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
     version="3.0">
 
+    <context-param>
+        <param-name>logbackDisableServletContainerInitializer</param-name>
+        <param-value>true</param-value>
+    </context-param>
+
+    <listener>
+        <listener-class>ch.qos.logback.classic.servlet.LogbackServletContextListener</listener-class>
+    </listener>
```

Comment:
Do you write in the order of `listener`,` context-param`? 
Do you write in the order of `context-param`,` listener`?
It looks like it's not unified.

Reply:
db18cc216024ac79cc5eb74c07ad6f9b4d00b5e8
Sorted by listener, context-param, according to the original order.

Reply:
- [ ] The behavior may change by changing the registration order of listeners. Is this taken into consideration?
At least to resolve this ISSUE, you must first register `LogbackServletContextListener`.